package.json

```diff
@@ -15,6 +15,9 @@
     "mongodb": "^3.0.1",
     "request": "^2.85.0",
     "simple-odata-server": "1.0.0",
-    "simple-odata-server-mongodb": "^1.0.0"
+    "simple-odata-server-mongodb": "^1.0.0",
```

Comment:
I know it was like this but remove the version range characters. It is not a good practice